src/current/v24.3/ldap-authorization.md

```diff
@@ -0,0 +1,113 @@
+---
+title: Configure LDAP Authorization
+summary: Learn how to configure role-based access control (authorization) using LDAP with directory services like Active Directory.
+toc: true
+---
+
+LDAP authorization allows CockroachDB to use group memberships accessed using LDAP, for example, from an Active Directory server, as the source of truth for [role](https://www.cockroachlabs.com/docs/stable/security-reference/authorization#roles) assignments. When enabled:
+
+1. During login, CockroachDB retrieves the user's group memberships using LDAP.
```

Comment:
This could use a similar set of edits to the suggestions in the Authentication page. Maybe something like:

1. When a client connects to the cluster using LDAP, the cluster looks up the user's group membership in the LDAP service.
2. Each LDAP group is mapped to a cluster role using the group's Common Name (CN) in the LDAP service.
3. The user is granted each corresponding role.
4. Roles that are no longer granted are revoked automatically.

(Is it really in this order, grants before revokes?)